databricks/koalas/frame.py

```diff
@@ -2626,7 +2657,11 @@ def describe(self) -> 'DataFrame':
         if len(exprs) == 0:
             raise ValueError("Cannot describe a DataFrame without columns")
 
-        sdf = self._sdf.select(*exprs).describe()
+        formatted_perc =  ["{:.0%}".format(p) for p in percentiles]
```

Comment:
I'd check if percentiles are in the range [0 1] otherwise, I raise an exception

Reply:
I am checking that in the new version